lib/elements/atom/install-statusbar-element.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+const CompositeDisposable = require('../../install/composite-disposable');
+
+class InstallStatusBarElement extends HTMLElement {
+  constructor(name) {
+    super();
+    this.name = name;
+    this.className = 'kite-install-status';
+    this.classList.add('inline-block');
+    this.subscriptions = new CompositeDisposable();
+    this.currentStatusText = '';
+  }
+
+  destroyTile() {
+    if (this.subscriptions) {
+      this.subscriptions.dispose();
+      this.subscriptions = null;
+    }
+    if (this.statusBarTile) {
+      this.statusBarTile.destroy();
+      this.statusBarTile = null;
+    }
+  }
+
+  init(install, statusBar) {
+    this.subscriptions.add(atom.tooltips.add(this, {
+      title: 'Kite will start automatically after being installed'
+    }));
+
+    this.statusBarTile = statusBar.addRightTile({item: this});
+    this.subscriptions.add(install.on('encountered-fatal-error', (err) => {
+      console.log('fatal status bar error: ' + err);
+      this.destroyTile();
+    }));
+
+    this.subscriptions.add(install.onDidFailStep(({error}) => {
+      console.log('fatal status bar error:' + (error && error.message));
+      this.destroyTile();
+    }));
+
+    this.subscriptions.add(install.observeState((state) => {
+      let newStatusText;
+      if (state.download) {
+        newStatusText = 'Downloading Kite';
+      }
+
+      if ((state.download && state.download.done) || state.install) {
+        newStatusText = 'Installing Kite';
+      }
+
+      if (state.running) {
+        newStatusText = 'Starting Kite';
+      }
+
+      if (state.plugin) {
+        if (state.plugin.done) {
+          this.destroyTile();
+          newStatusText = '';
+        } else {
+          newStatusText = 'Installing Kite plugin';
```

Comment:
What's the difference between this status and the status with text 'Installing Kite'? Should they have the same status text, or more of a distinction if they're supposed to be different?